File: 867-transpose-matrix/867-transpose-matrix.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& matrix) {
        int m = matrix.size();
        int n = matrix[0].size();
        vector<vector<int>>transpose(n,vector<int>(m));
        for(int i=0;i<n;i++){
            for(int j=0;j<m;j++){
                transpose[i][j] = matrix[j][i];
            }
        }
        return transpose;
    }
};

File: 867-transpose-matrix/867-transpose-matrix.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        m = len(matrix)
        n = len(matrix[0])
        transpose = []
        for i in range(0,n):
            temp = []
            for j in range(0,m):
                temp.append(matrix[j][i])
            transpose.append(temp)
        return transpose
        